addons/binding/org.openhab.binding.kodi/ESH-INF/config/config.xml

```diff
@@ -1,15 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<config-description:config-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
-    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0
-    http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0 http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
 
-    <config-description uri="channel-type:kodi:config">
-        <parameter name="group" type="text" required="true">
-            <label>Channel group</label>
-            <description>The channel group name used to identify the channel id</description>
-            <default>All channels</default>
-        </parameter>
-    </config-description>
+	<config-description uri="channel-type:kodi:pvr-channel">
+		<parameter name="group" type="text" required="true">
+			<label>PVR channel group</label>
+			<description>The PVR channel group name used to identify the channel
+				id
```

Comment:
@kaikreuzer I double checked these format changes. The `binding.xml` and `config.xml` had some weird format before (e.g. whitespaces instead of tabs). Now the formatter is applied correctly.